EmailApi/Email.Api/Services/MailService.cs

```
@@ -11,30 +11,39 @@ public void SendEmail(MailRequest mailRequest)
         {
             try
             {
+                // Mistake 1: Hardcoded SMTP server and port instead of using config
                 SmtpClient smtpServer = new SmtpClient("in-v3.mailjet.com", 587)
                 {
                     EnableSsl = true,
                     UseDefaultCredentials = false,
-                    Credentials = new NetworkCredential("745752d3e4dbaf360f9b83da4e7577ac", "8747b2b74209ab9144a7a4b9111cecb3"),
```

Recipient email address is used without format or null validation. Add null/empty and email format validation to fail fast and avoid runtime exceptions and potential abuse.